packages/perspective-workspace/src/ts/workspace/workspace.ts

```diff
@@ -112,6 +236,10 @@ export class PerspectiveWorkspace extends SplitPanel {
     }
 
     init_indicator() {
+        const exists = document.querySelector("body > perspective-indicator");
+        if (exists) {
+            return exists as HTMLElement;
+        }
```

Comment:
This smells like a previous `<perspective-workspace>` that was disconnected leaked this element, not the responsibility of this method to correct.